source/MaterialXGenGlsl/GlslSyntax.cpp

```diff
@@ -94,6 +94,7 @@ namespace
     };
 }
 
+const string GlslSyntax::BLOCK_VARIABLE_SEPARATOR = ";\n";
```

Comment:
Could we make this a member of ShaderGenerator instead and name it SEMICOLON_NEWLINE.